src/lmql/runtime/langchain_lmtp.py

```diff
@@ -0,0 +1,336 @@
+"""
+Langchain interface to LMTP models.
+"""
+from typing import TYPE_CHECKING, Any, Dict, List, Optional, Union
+
+import logging
+
+import json
+import asyncio
+from weakref import WeakValueDictionary
+import aiohttp
+
+from langchain.llms.base import LLM
+from langchain.callbacks.manager import (
+    AsyncCallbackManagerForLLMRun,
+    CallbackManagerForLLMRun,
+)
+from langchain.llms.utils import enforce_stop_tokens
+from langchain.schema import LLMResult
```

Comment:
Is langchain already in the dependency tree? If not, do we want to make it an extra in `setup.cfg`?

Reply:
I think for this case it is not really necessary. After all, users that want to use LMTP in LC, must LC already installed. Same for `.aschain()`.